package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Authenticator - Handles authentication and generates user token",
   "main": "index.js",
   "scripts": {
+    "build": "tsc && cp src/authentication/metadata.xml dist/authentication/metadata.xml",
```

Comment:
Either put metadata.xml at the root level of the project, as it should be, or at least separate to `"build:tsc": "tsc" and "build:assets": "cp src/authentication/metadata.xml dist/authentication/metadata.xml"`.

Reply:
Also delete `.gitlab-ci.yml`.

Reply:
Why name the scripts differently? And you should use them in script `build` like this: `"build": "npm run build:tsc && npm run build:assets"`